RELEASE-NOTES.md

```diff
@@ -8,6 +8,16 @@ Latest release:
 
 ## Announcement
 
+### Private key renewal
+
+This realease turns on an important security feature: a new private key will be now created every 30 days by default.
+Existing sealed-secrets resources will still be decrypted until the keys are manually phased out.
+
+You can read more about this feature in this [README section](https://github.com/bitnami-labs/sealed-secrets#key-renewal) or in the original GH issue #137.
+
+This feature impacts people who use the offline workflow with `kubeseal --cert`, who will have to update
+the certificates more often.
```

Comment:
I think this deserves more clarification than «more often», as technically the validity of the saved cert would depend on the previous/old privkeys garbage collection that people would implement by themselves (something that would also need to be documented/hinted about IMO).

Reply:
actually I had a brain fart, good catch